src/main/java/org/agbrothers/englishbot/processing/processor/lesson/LessonProcessor.java

```diff
@@ -31,10 +32,14 @@ public void process(ProcessingExchange exchange) {
         Lesson lesson = lessonService.getLesson(exchange.getUser());
         String messageText = exchange.getMessageText();
 
+        ResponseMessage responseMessage = new ResponseMessage();
+        exchange.getResponseMessages().add(responseMessage);
```

Comment:
I assume you add this `responseMessage` to the list because of logic in `appendResponseMessageText`. Please leave a TODO comment here to fix this within https://agbrothers.atlassian.net/browse/EN-67

Reply:
Ok, left comment